.golangci.yml

```diff
@@ -18,6 +18,7 @@ linters:
     - lll
     - misspell
     - nakedret
+    - nolintlint
```

Comment:
Oh! Didn't know about this linter; looks handy. Perhaps I should enable it one some other repositories